src/lib/button-toggle/button-toggle.html

```diff
@@ -5,10 +5,14 @@
          [checked]="checked"
          [disabled]="disabled"
          [name]="name"
+         (blur)="_onInputBlur()"
+         (focus)="_onInputFocus()"
          (change)="_onInputChange($event)"
          (click)="_onInputClick($event)">
 
   <div class="mat-button-toggle-label-content">
     <ng-content></ng-content>
   </div>
 </label>
+<!-- the touchstart handler prevents the overlay from capturing the initial tap on touch devices -->
+<div class="mat-button-toggle-focus-overlay" (touchstart)="$event.preventDefault()"></div>
```

Comment:
This is the same as https://github.com/angular/material2/blob/master/src/lib/button/button.html